src/lib/y2network/connection_config/wireless.rb

```diff
@@ -26,7 +26,7 @@ class Wireless < Base
       # wireless options
       #
       # FIXME: Consider an enum
-      # @return [String] (Ad-hoc, Managed, Master)
+      # @return [Symbol] (:ad-hoc, :managed, :master)
```

Comment:
NP: `:ad_hoc`

Reply:
It is not just a NP, the interface_file , does a only a downcase.to_sym, which is fine when reading but as you commented we need an underscore substitution and do the inverse operation when writing the file. 

So, by now will maintin the original string and just ensure we are handling it correctly

Reply:
Ah! right!

I simply thought that you make a typo in the comment, but it didn't come to my mind the described problem while I was doing the review.

Good point!